ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -0,0 +1,564 @@
+    'ceph_osd_blocked-by_--format_json-pretty',
+    'ceph_osd_crush_class_ls_--format_json-pretty',
+    'ceph_osd_crush_dump_--format_json-pretty',
+    'ceph_osd_crush_rule_dump_--format_json-pretty',
+    'ceph_osd_crush_rule_ls_--format_json-pretty',
+    'ceph_osd_crush_show-tunables_--format_json-pretty',
+    'ceph_osd_crush_weight-set_dump_--format_json-pretty',
+    'ceph_osd_crush_weight-set_ls_--format_json-pretty',
+    'ceph_osd_df_--format_json-pretty',
+    'ceph_osd_df_tree_--format_json-pretty',
+    'ceph_osd_dump_--format_json-pretty',
+    'ceph_osd_getmaxosd_--format_json-pretty',
+    'ceph_osd_lspools_--format_json-pretty',
+    'ceph_osd_numa-status_--format_json-pretty',
+    'ceph_osd_perf_--format_json-pretty',
+    'ceph_osd_pool_ls_detail_--format_json-pretty',
+    'ceph_osd_stat_--format_json-pretty',
+    'ceph_osd_tree_--format_json-pretty',
+    'ceph_osd_utilization_--format_json-pretty',
+    'ceph_pg_dump_--format_json-pretty',
+    'ceph_pg_stat_--format_json-pretty',
+    'ceph_progress_--format_json-pretty',
+    'ceph_progress_json',
+    'ceph_progress_json_--format_json-pretty',
+    'ceph_quorum_status_--format_json-pretty',
+    'ceph_report_--format_json-pretty',
+    'ceph_service_dump_--format_json-pretty',
+    'ceph_status_--format_json-pretty',
+    'ceph_time-sync-status_--format_json-pretty',
+    'ceph_versions_--format_json-pretty',
+]
+
+GATHER_COMMANDS_OTHERS = [
+    'buildconfigs.yaml', 'builds.yaml', 'configmaps.yaml', 'cronjobs.yaml',
+    'daemonsets.yaml', 'db-noobaa-db-0.yaml', 'deploymentconfigs.yaml',
+    'deployments.yaml', 'endpoints.yaml', 'events.yaml',
+    'horizontalpodautoscalers.yaml', 'imagestreams.yaml', 'jobs.yaml',
+    'my-alertmanager-claim-alertmanager-main-0.yaml',
+    'my-alertmanager-claim-alertmanager-main-1.yaml',
+    'my-alertmanager-claim-alertmanager-main-2.yaml',
+    'my-prometheus-claim-prometheus-k8s-0.yaml',
+    'my-prometheus-claim-prometheus-k8s-1.yaml',
+    'noobaa-core-0.log', 'noobaa-core-0.yaml',
+    'noobaa-db-0.log', 'noobaa-db-0.yaml', 'noobaa-default-backing-store.yaml',
+    'noobaa-default-bucket-class.yaml', 'noobaa.yaml',
+    'ocs-storagecluster-cephblockpool.yaml', 'ocs-storagecluster-cephcluster.yaml',
+    'ocs-storagecluster-cephfilesystem.yaml', 'openshift-storage.yaml',
+    'persistentvolumeclaims.yaml', 'pods.yaml', 'pools_rbd_ocs-storagecluster-cephblockpool',
+    'previous.log', 'previous.log', 'registry-cephfs-rwx-pvc.yaml', 'replicasets.yaml',
+    'replicationcontrollers.yaml', 'rook-ceph-mon-a.yaml', 'rook-ceph-mon-b.yaml',
+    'rook-ceph-mon-c.yaml', 'routes.yaml', 'secrets.yaml', 'services.yaml',
+    'statefulsets.yaml', 'storagecluster.yaml',
+]
+
+GATHER_COMMANDS_CEPH_4_5 = [
+    'ceph_osd_blacklist_ls',
+]
+
+GATHER_COMMANDS_JSON_4_5 = []
+
+GATHER_COMMANDS_OTHERS_4_5 = []
+
+GATHER_COMMANDS_CEPH_4_6 = []
+
+GATHER_COMMANDS_JSON_4_6 = []
+
+GATHER_COMMANDS_OTHERS_4_6 = [
```

Comment:
https://github.com/openshift/ocs-operator/blob/fefc8a0e04e314801809df2e5292a20fae8456b8/must-gather/collection-scripts/gather_namespaced_resources#L36-L46

https://github.com/openshift/ocs-operator/blob/fefc8a0e04e314801809df2e5292a20fae8456b8/must-gather/collection-scripts/gather_namespaced_resources#L50-L54

https://github.com/openshift/ocs-operator/blob/fefc8a0e04e314801809df2e5292a20fae8456b8/must-gather/collection-scripts/gather_namespaced_resources#L58-L62

https://github.com/openshift/ocs-operator/blob/fefc8a0e04e314801809df2e5292a20fae8456b8/must-gather/collection-scripts/gather_clusterscoped_resources#L17-L23

https://github.com/openshift/ocs-operator/blob/fefc8a0e04e314801809df2e5292a20fae8456b8/must-gather/collection-scripts/gather_clusterscoped_resources#L27-L33 

same for yaml also ^

Reply:
noobaa cli add noobaa file name=`status`

tar file noobaa_diagnostics_2020-11-04T08:24:09Z.tar.gz
be carefull time is also added in the name.